### PR TITLE
User model

### PR DESCRIPTION
**Pull request by eytienne87**
@vanthaohuynh Can you review the user model and merge after?

### CODE REVIEW

**Comment by eytienne87** [comment]
I still need to adjust something wait before merging

**Comment by eytienne87** [comment]
@vanthaohuynh this should be complete please and review and merge if satisfactory

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+
```
Comment (vanthaohuynh) [inline app/models/user.rb:2]: Should remove the blank line (number 2)

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  validates :first_name, :last_name, presence: true
```
Comment (vanthaohuynh) [inline app/models/user.rb:3]: Should have an empty line after line 3

**Review by vanthaohuynh — COMMENTED** [review]
Look all good, except the 2 styling issues, see inline comments in user.rb

**Comment by eytienne87** [comment]
@vanthaohuynh styling issues fixed, please merge

**Pull request merged**